src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,11 +1,24 @@
 package core.basesyntax.service;
 
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int VALID_AGE = 18;
+    private static final int VALID_PASSWORD_LENGTH = 6;
+    private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null
+                || user.getAge() == null
+                || user.getAge() < VALID_AGE
+                || user.getLogin() == null
+                || user.getPassword() == null
+                || user.getPassword().length() < VALID_PASSWORD_LENGTH) {
+            throw new RuntimeException("Invalid data entered. :(");
```

Comment:
where is the check for this?
<img width="408" alt="Screenshot 2021-11-16 at 14 52 04" src="https://user-images.githubusercontent.com/60258373/141988606-c97430f2-44a8-4b42-8987-33da2cbb92c2.png">

Reply:
Add